saw-remote-api/python/pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = ["Andrew Kent <andrew@galois.com>", "Aaron Tomb <atomb@galois.com>"]
 python = "^3.8"
 requests = "^2.25.1"
 BitVector = "^3.4.9"
-cryptol = { path = "../../deps/cryptol/cryptol-remote-api/python/", develop = true }
+cryptol = "2.11.0"
```

Comment:
How did version 2.11.0 make it to PyPI before GaloisInc/cryptol#1135 landed?

Reply:
There is no automation/CI/etc inherently linking the cryptol repo and that PR to the PyPI upload/package. I think changing that so it is more automated and less manual would be an improvement, but it hasn't been implemented yet.

In other words, I pushed the PyPI package before merging that PR so I could further test and check things had worked out as expected, with the thought that if there was a needed change it would become a part of that PR and then a subsequent PyPI package patch/update.

Reply:
Gotcha. I suppose I was confused since I associate PyPI releases with `git` tags, but now that I check, there doesn't seem to be a convention that a given PyPI release of `cryptol` corresponds to a particular tag in the `cryptol` repo.